it/endpoints/CreateTestBusinessControllerISpec.scala

```diff
@@ -79,17 +80,26 @@ class CreateTestBusinessControllerISpec extends UnitSpec with IntegrationBaseSpe
 
   "Calling the create Business endpoint" should {
     "return a 201 status code" when {
-      "a valid request is made" in new Test {
+      "a valid request is made with a self-employment business" in new Test {
+        val response: WSResponse = await(request().post(MinimalCreateTestBusinessRequest.SelfEmployment.mtdBusinessJson))
 
-        val response: WSResponse = await(request().post(MinimalCreateTestBusinessRequest.mtdBusinessJson))
+        response.status shouldBe CREATED
+        response.header("X-CorrelationId") should not be empty
+        response.json shouldBe Json.parse(expectedResponseBody)
+      }
+
+      "a valid request is made with a uk-property business" in new Test {
```

Comment:
What was the motive for having two different tests for a self-employment and a property business? Would the validator unit tests not suffice for testing all the logic for the way?

Reply:
I think it's because I couldn't decide which one to use. 😂 Do you have any preferences for one? They run through different validators.

Reply:
I would personally think that if it is only the validations which change depending on the type of business (i.e. there is no other module e.g. a service which does different things depending on the type) then I think those UTs for the validator would be enough and integration testing it wouldn't add much more benefit. Would you agree?

Reply:
I made the change.

https://github.com/hmrc/mtd-sa-test-support-api/pull/66/commits/88e08f1cd153c159baea8045bce952311fdbf856